### ISSUE TITLE
Check item 288

### ISSUE THREAD

**Issue by bast** [constructicon/russian-data#4]
> Also, there seems to be a problem with this item:
> 288
> NAME NP (размером/величиной) с NP-Acc - Собака была размером с лошадь.
> It generates two menu items in Morphology, one is: NP (размером, and the other is: величиной) с NP.
> Maybe the / messed it up?
> And there are many items with the label “NA”, I don't understand how those work.

**Comment by bast** [comment]
Also this type of problems will go away once we move away from the spreadsheet and only edit YAML files. This problem arose because I need to convert spreadsheet -> csv -> yaml.

**Comment by Labrus** [comment]
I have fixed this in the database, this was my mistake. All NA items are now replaced with tags, 288 item is fixed, the words appeared in that slot by mistake and mixed with tags. This problem arose because I was asked to provide the database as I have it concerning the syntactic and morphological annotation, and some tags are there not updated, this leads to duplicate tags with and without capital letters, etc. I can fix these inconsistencies in the database in the beginning of January if I can only focus on them.
